pkg/commands/sso/root.go

```diff
@@ -188,6 +219,55 @@ func (c *RootCommand) identifyProfileAndFlow() (profileName string, flow Profile
 	}
 }
 
+func (c *RootCommand) processCustomer(ar auth.AuthorizationResult) error {
+	debugMode, _ := strconv.ParseBool(c.Globals.Env.DebugMode)
+	apiEndpoint, _ := c.Globals.APIEndpoint()
+	data, err := undocumented.Call(undocumented.CallOptions{
+		APIEndpoint: apiEndpoint,
+		HTTPClient:  c.Globals.HTTPClient,
+		HTTPHeaders: []undocumented.HTTPHeader{
+			{
+				Key:   "Accept",
+				Value: "application/json",
+			},
+			{
+				Key:   "User-Agent",
+				Value: useragent.Name,
+			},
+		},
+		Method: http.MethodGet,
+		Path:   "/verify",
```

Comment:
I'd personally suggest using `/current_customer` instead, which has less data than than `/verify` and the field's you need `id` and `name` in the response. 

https://www.fastly.com/documentation/reference/api/account/customer/#get-logged-in-customer